dlt/common/normalizers/json/relational.py

```diff
@@ -96,7 +96,7 @@ def _reset(self) -> None:
         # self.primary_keys = Dict[str, ]
 
     def _flatten(
-        self, table: str, dict_row: DictStrAny, _r_lvl: int
+        self, table: str, dict_row: DictStrAny, parent_path: Tuple[str, ...], _r_lvl: int
```

Comment:
this is probably the most often called function in our code. maybe we can skip passing additional argument here or do something else to make it faster?
- count the recursion level down (r_lvl) with the initial value initialized to max nesting per table. so we do not unnest when  below _r_lvl
- pass just parent table name, should be faster than tuple